app/views/links/_table.html.erb

```diff
@@ -11,6 +11,7 @@
             <th>Page name</th>
             <th>Description</th>
             <th></th>
+            <th></th>
```

Comment:
Itt légyszi ne vezessünk be egy új oszlopot minden gombnak. Ha meg tudjuk oldani, akkor maradjanak egy oszlopban, és az adott cellán belül rendezzük majd őket egy sorba.
```html
<td>
  <div class="feltehetőleg-display-flex">
    <%= link_to '' %>
    <%= link_to '' %>
  </div>
</td>
```